src/Sylius/Bundle/AdminBundle/Resources/views/AdminUser/_form.html.twig

```diff
@@ -26,6 +26,11 @@
                 <label>{{ 'sylius.ui.avatar'|trans }}</label>
                 {{ form_row(form.avatar, {'label': false}) }}
             </div>
+            {% if event_prefix == 'sylius.admin.admin_user.update' %}
```

Comment:
:( Do we want to show this part only on update action? This solution won't work if we rename route